lib/sioFb.js

```diff
@@ -49,13 +51,18 @@ class SioFb {
   };
 
   getNewStrips() {
-    return this.getLatestStrips()
+    let lastPolledDate;
+    console.log('Polling for new strips...');
+    return this.repository.getLastPolledDate()
+      .selectMany(date => {
+        console.log('Polling last run on %s.', date);
+        lastPolledDate = date;
+        return this.getLatestStrips();
```

Comment:
also this feels like it's a private method, you can achieve it declaring is as a `function()` inside the scope and you won't have it exposed.

Not sure if that's still a thing in es6, it was when you had to declare classes in es5